KubeArmor/eventAuditor/eventAuditor.go

```diff
@@ -1,12 +1,14 @@
 package eventAuditor
 
+import fd "github.com/kubearmor/KubeArmor/KubeArmor/feeder"
+
 // =================== //
 // == Event Auditor == //
 // =================== //
 
 // EventAuditor Structure
 type EventAuditor struct {
-	//
+	LogFeeder *fd.Feeder
```

Comment:
we will eventually use the log feeder later, but we don't need to think about it at this point.

Reply:
I see. I added it once we must to stick to an error checking and logging logic, like comment below. So I'll maintain this change for now. That's ok?